NUnit.sln.DotSettings

```diff
@@ -42,8 +42,22 @@
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=TypeParameters/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="T" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=TypesAndNamespaces/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb_AaBb"&gt;&lt;ExtraRule Prefix="" Suffix="" Style="AA_BB" /&gt;&lt;/Policy&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/UserRules/=03c58a20_002Df24c_002D4992_002Da01b_002Dd4dd88b40443/@EntryIndexedValue">&lt;Policy&gt;&lt;Descriptor Staticness="Static" AccessRightKinds="Private, Internal" Description="Non-visible static readonly and constant fields"&gt;&lt;ElementKinds&gt;&lt;Kind Name="READONLY_FIELD" /&gt;&lt;Kind Name="CONSTANT_FIELD" /&gt;&lt;/ElementKinds&gt;&lt;/Descriptor&gt;&lt;Policy Inspect="False" Prefix="" Suffix="" Style="AaBb" /&gt;&lt;/Policy&gt;</s:String>
+	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/UserRules/=15b5b1f1_002D457c_002D4ca6_002Db278_002D5615aedc07d3/@EntryIndexedValue">&lt;Policy&gt;&lt;Descriptor Staticness="Static" AccessRightKinds="Private" Description="Static readonly fields (private)"&gt;&lt;ElementKinds&gt;&lt;Kind Name="READONLY_FIELD" /&gt;&lt;/ElementKinds&gt;&lt;/Descriptor&gt;&lt;Policy Inspect="False" Prefix="" Suffix="" Style="AaBb" /&gt;&lt;/Policy&gt;</s:String>
```

Comment:
ReSharper should be able to use our .editorconfig which has the naming rules in it: https://www.jetbrains.com/help/resharper/Using_EditorConfig.html
So there should be no need for naming conventions in this file.

Reply:
First, this file should be in a separate PR, not here , but slipped through.

Do we want Resharper/Rider to write into the editorconfig with their specific (more finegrained) code and style rules?  
E.g.
```
# ReSharper custom properties for code formatting styles
resharper_csharp_brace_style=next_line
resharper_csharp_blank_lines_around_invocable=2

# ReSharper custom properties for code syntax styles
csharp_default_private_modifier=explicit
braces_for_ifelse=not_required

# ReSharper custom properties for code inspections
resharper_possible_null_reference_exception_highlighting=error
resharper_replace_with_string_is_null_or_empty_highlighting=none
```

Or should we just keep them in the dotsettings file?

At the same time, I - for me - don't want to spend too much time bothering about these settings.  I haven't even looked into what these "new" settings actually are.

The upgrade of this file happened automagically when I upgraded to the latest Resharper version.

The point of having this file here is to allow devs using Rider or Resharper to work on the code.  I am non-opiniated on what tools people use.